features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -586,6 +586,7 @@
             <UserInfoEndpointClaimRetriever>org.wso2.carbon.identity.oauth.endpoint.user.impl.UserInfoUserStoreClaimRetriever</UserInfoEndpointClaimRetriever>
             <UserInfoEndpointRequestValidator>org.wso2.carbon.identity.oauth.endpoint.user.impl.UserInforRequestDefaultValidator</UserInfoEndpointRequestValidator>
             <UserInfoEndpointAccessTokenValidator>org.wso2.carbon.identity.oauth.endpoint.user.impl.UserInfoISAccessTokenValidator</UserInfoEndpointAccessTokenValidator>
+            <UserInfoInternalPrefixedRolesClaimAllowed>true</UserInfoInternalPrefixedRolesClaimAllowed>
```

Comment:
What about using a name like <UserInfoRemoveInternalPrefixFromRoles>? It seems clearer and directly describes the action of removing the "Internal" prefix from roles. 
The current name, UserInfoInternalPrefixedRolesClaimAllowed, feels more like it’s about not allowing internal prefixed roles at all, which might be misleading. WDYT?

Reply:
Fixed by - https://github.com/wso2/carbon-identity-framework/pull/6295/commits/69042ffdabe3a91ad689e45b95b9d62f24916382